.travis.yml

```
@@ -0,0 +1,21 @@
+dist: xenial
+language: python
+python:
+  - "3.7"
+  - "3.8"
+os:
+  - linux
+before_script:
+  - python -m pip install pip --upgrade
+  - python -m pip list
+  - export PYTHONHASHSEED=${PYTHONHASHSEED:-${RANDOM}}
+  - echo "export PYTHONHASHSEED=${PYTHONHASHSEED}"
+  - python -m pip install .[test]
+  - python -m pip install codecov
+  - export COVERAGE_PROCESS_START=$(pwd)/.coveragerc
+  - export PYTEST_ADDOPTS=-v
+script:
+  - python -m flake8
+  - python -m coverage run -m pytest
```

Check formatting as well. Might want to sync that against the commit hook.
```suggestion
script:
  - python -m flake8
  - python -m black
  - python -m coverage run -m pytest
```